### DISCUSSION TITLE
Por onde começar a estudar ES6?

### DISCUSSION THREAD

**Discussion by paulinhoerry** [frontendbr/forum#289] [Perguntas e Respostas]
Bom galera, devido a uma porrada de projetos que eu estava fazendo nos últimos meses, terminei por me atrasar em aprender ES6. Tenho uma boa noção de ES5 e agora pretendo migrar pro ES6 e estou precisando de um norte. Vocês poderiam me indicar livros ou cursos para me guiar?

**Comment by bukinoshita** [comment]
http://es6.io

**Comment by grodrigues** [comment]
Indico Também o curso de Flexbox do Wes Bos.
http://flexbox.io/

**Comment by renatogalvones** [comment]
@grodrigues talvez o curso de ES6 dele funcione melhor... rsrsrs

http://wesbos.com/es6-for-everyone/

**Comment by guilhermecomum** [comment]
Opa e ae blz?
Recomendo esses dois artigos para começar: 
- http://medium.com/@matheusml/o-que-todo-desenvolvedor-javascript-precisa-saber-2cc33daedb86
- https://medium.com/@matheusml/o-guia-do-es6-tudo-que-voc%C3%AA-precisa-saber-8c287876325f#.auny7n8v3

**Comment by felquis** [comment]
Este livro parece bacana http://exploringjs.com/es6/

**Comment by felquis** [comment]
E para ser Dev JS acho interessante passar pela série https://github.com/getify/You-Dont-Know-JS que tem muito conteúdo bom.

**Comment by camilatigre** [comment]
Eu li [esse livro](https://leanpub.com/understandinges6/read) e vi [esses videos](https://www.youtube.com/watch?v=LTbnmiXWs2k&list=PL57atfCFqj2h5fpdZD-doGEIs0NZxeJTX). Recomendo ((:

**Comment by bernardodiasc** [comment]
- https://ponyfoo.com/articles/es6 artigos muito bons
- http://es6-features.org/ referencias comparando com es5
